node/src/test/test-ActiveSpeakerObserver.ts

```diff
@@ -27,8 +27,14 @@ beforeEach(async () => {
 	ctx.router = await ctx.worker.createRouter({ mediaCodecs: ctx.mediaCodecs });
 });
 
-afterEach(() => {
+afterEach(async () => {
 	ctx.worker?.close();
+
+	if (ctx.worker?.subprocessClosed === false) {
```

Comment:
Maybe early return? 

```suggestion
	if (ctx.worker?.subprocessClosed) {
	    return;
	}
```

Reply:
There is a comment above why `=== false` is necessary here

Reply:
> There is a comment above why === false is necessary here

I'm suggesting to early return if subprocess is closed so the following block (awaiting for it) is not indented. AFAIK we extensively use this pattern.

Reply:
This is not appropriate here IMHO. This is not a method in which if some condition is false we don't want to do anything. See test-DirectTransport (or maybe test-PipeTransport) in which we create 2 workers so this early return fashion cannot be done because we have to do the same for both workers independently. This is not like checking the this.closed flag in a close() method. If I apply your suggestion and tomorrow we need to add more cleanup code in those afterEach() hooks then I would need to revert those changes  and leave the code as it is now in this PR.

Reply:
Also your suggested code is wrong @jmillan. If the worker.subprocessClosed is not true it doesn't mean that we have to await for an event to happen in that worker, because as explained above the worker MAY NOT exist.

Reply:
Good